### PR TITLE
[All hosts] (browser) using more familiar terminology in browser article

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b57a5ce](https://github.com/OfficeDev/office-js-docs-pr/pull/1032/commits/b57a5ce432de66446abc2db643dbcaeee7ef1812)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/concepts/requirements-for-running-office-add-ins.md) | :warning:Warning | | [Details](#user-content-12ff82a2-7e1bf13b)
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="12ff82a2-7e1bf13b"></a>
### [docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/concepts/requirements-for-running-office-add-ins.md)
 - **Line 46**: **[Warning]** `````Invalid file link:(~/concepts/web-viewers-used-by-office-web-add-ins.md).`````
 - **Line 88**: **[Warning]** `````Invalid file link:(~/concepts/web-viewers-used-by-office-web-add-ins.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905232226572331-1032%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gK73cKl9LYvMu3U1Y3YRR6epivRCJSiwsXpCEslkgBI%3D&st=2019-05-23T22%3A23%3A22Z&se=2019-06-23T22%3A28%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0b5adca](https://github.com/OfficeDev/office-js-docs-pr/pull/1032/commits/0b5adca8541b7d81f8c549eaffb8a3d931162ef0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905232236083360-1032%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OzfEDvcNieh90%2BgyAZdEjSzG7%2FfxRBrD%2FzOtMR6pINs%3D&st=2019-05-23T22%3A32%3A29Z&se=2019-06-23T22%3A37%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Rick-Kirkham** [comment]
Alex didn't get to this before he left, so adding @kbrandl as reviewer.

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-ms.date: 05/03/2019
+ms.date: 05/23/2019
 localization_priority: Priority
 ---
 
-# Web viewers used by Office Add-ins
+# Browsers used by Office Add-ins
 
-Since Office Add-ins are web applications, they need a web page viewer to display the HTML pages of the web application and a JavaScript engine to run the JavaScript. Both are supplied by a browser installed on the user’s computer.
+Since Office Add-ins are web applications, they need an embedded mini-browser to display the HTML pages of the web application in a task pane or content add-in window. (An embedded mini-browser is typically called a WebView control or WebBrowser control.) Add-ins also need a JavaScript engine to run the JavaScript. Both the mini-browser and the engine are supplied by a browser installed on the user’s computer.
```
Comment (kbrandl) [inline docs/concepts/browsers-used-by-office-web-add-ins.md]: I'm not sure what "mini-browser" means (as opposed to just "browser"). Suggest that you consider removing "mini-", if it's accurate to just say "browser".

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Web viewers used by Office Add-ins
+title: Browsers used by Office Add-ins
 description: ''
-ms.date: 05/03/2019
+ms.date: 05/23/2019
```
Comment (kbrandl) [inline docs/concepts/browsers-used-by-office-web-add-ins.md]: Suggest that you populate the value for the `description` attribute, as doing so improves SEO.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[291147b](https://github.com/OfficeDev/office-js-docs-pr/pull/1032/commits/291147b501a603fd517a5ac7ea6725e78145390a)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/.openpublishing.redirection.json) | :x:Error | | [Details](#user-content-10516278-2a2c64d6)

<a id="10516278-2a2c64d6"></a>
### [.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/.openpublishing.redirection.json)
 - **[Error]** `````source_path '/office/dev/add-ins/concepts/web-viewers-used-by-office-web-add-ins' in master redirection file should point to a file of supported extension: .md, .yml.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905281938005870-1032%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FLBL4uhJ%2BjVmrJKo3tZ5V8kW2eQIcLFq%2F3vdytIpf0c%3D&st=2019-05-28T19%3A33%3A35Z&se=2019-06-28T19%3A38%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8dffad3](https://github.com/OfficeDev/office-js-docs-pr/pull/1032/commits/8dffad3196edba7dcff220f31d0c1239a904b77b)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/.openpublishing.redirection.json) | :x:Error | | [Details](#user-content-10516278-9086036a)

<a id="10516278-9086036a"></a>
### [.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/.openpublishing.redirection.json)
 - **[Error]** `````source_path '/office/dev/add-ins/concepts/web-viewers-used-by-office-web-add-ins.md' is above the root path 'W:\m3c1-s'.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905281940371722-1032%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=sPUAynzjzjxGy5QXoTNHm0PAy76YnplCpF2WiIwPkxk%3D&st=2019-05-28T19%3A36%3A07Z&se=2019-06-28T19%3A41%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: .openpublishing.redirection.json [hunk 3]
```diff
@@ -101,6 +101,10 @@
         "redirect_url": "/office/dev/add-ins/concepts/add-in-development-best-practices", 
         "redirect_document_id": true 
       }, 
+      {   
+        "source_path": "/office/dev/add-ins/concepts/web-viewers-used-by-office-web-add-ins",
```
Comment (kbrandl) [inline .openpublishing.redirection.json]: The value of the `source_path` attribute needs to end with `.md`

File: .openpublishing.redirection.json [hunk 4]
```diff
@@ -101,6 +101,10 @@
         "redirect_url": "/office/dev/add-ins/concepts/add-in-development-best-practices", 
         "redirect_document_id": true 
       }, 
+      {   
+        "source_path": "/office/dev/add-ins/concepts/web-viewers-used-by-office-web-add-ins.md",
```
Comment (kbrandl) [inline .openpublishing.redirection.json]: Sorry, one more comment -- seems like source path should be:

`docs/concepts/web-viewers-used-by-office-web-add-ins.md`

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1c7f163](https://github.com/OfficeDev/office-js-docs-pr/pull/1032/commits/1c7f163f93555eb4054064a089d008dd4e7c8368)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/webview/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905281947432118-1032%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=RTlYkUr%2B6t0ZKJzePJkKSxuhPiytZv7Q3oopDWlMZoQ%3D&st=2019-05-28T19%3A44%3A21Z&se=2019-06-28T19%3A49%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**